modules/pubmaticBidAdapter.js

```diff
@@ -737,6 +739,44 @@ function _addFloorFromFloorModule(impObj, bid) {
   impObj.bidfloor = ((!isNaN(bidFloor) && bidFloor > 0) ? bidFloor : UNDEFINED);
 }
 
+function _populateSegmentDataIfAvailable(fpdUserData, segmentData) {
+  // fpdUserData - data set from global and bidder level configs
+  // segmentData - data set from permutive
+  var dataMerged = false;
+
+  if ((!fpdUserData || !fpdUserData.data) && (!segmentData || segmentData.length === 0)) {
+    return;
+  }
+
+  if ((!fpdUserData || !fpdUserData.data) && segmentData.length > 0) {
```

Comment:
If fpdUserData is set but segmentData is not, this line throws an exception.

I saw this exception testing the the example pbjs.setConfig() from the First Party Data page:
https://docs.prebid.org/features/firstPartyData.html

<img width="1242" alt="Screen Shot 2021-06-23 at 11 45 06 AM" src="https://user-images.githubusercontent.com/13356910/123151655-8aeca400-d418-11eb-80c9-651d2492cd8b.png">

Reply:
This has been fixed